src/driver/main.cpp

```diff
@@ -38,6 +38,7 @@
 #include <migraphx/json.hpp>
 #include <migraphx/version.h>
 
+#include <migraphx/instruction.hpp>
```

Comment:
Why do we need an extra include for instruction here?

Reply:
The compilation on Windows is failing because of incomplete type `instruction`.